src/components/modal/_modal.scss

```diff
@@ -0,0 +1,57 @@
+.modal {
+    display: none; 
+    position: fixed; 
+    z-index: 200; 
+    left: 0;
+    top: 0;
+    width: 100%; 
+    height: 100%; 
+    overflow: auto; 
+    background-color: rgb(31,31,31); 
+    background-color: rgba(31,31,31,0.75); 
+
+  
+    .modal-content {
+        margin: 15% auto; 
+        padding: 40px;
+        width: 420px;	
+        background-color: #FFFFFF; 
+        position: relative;
+
+        h2 {
+            color: #000000;	
```

Comment:
Couple of points:

- We've been using Sass variables for the colours, so instead of `#000` can you use `$black`?
- There are font-size variables as well for set sizes
- as for defaults, the font-family should already be defaulting to `Verdana` without needing to be set again. Same goes for the 16px in the `<p>` tag -- this should already be the default size.